config/initializers/1_settings.rb

```diff
@@ -125,6 +123,20 @@ def verify_constant(modul, current, default)
 Settings.gitlab['repository_downloads_path'] = File.absolute_path(Settings.gitlab['repository_downloads_path'] || 'tmp/repositories', Rails.root)
 
 #
+# Outgoing emails
+#
+Settings['outgoing_emails'] ||= Settingslogic.new({})
+Settings.outgoing_emails['enabled']           ||= Settings.gitlab['email_enabled'] unless Settings.gitlab['email_enabled'].nil?  # for backward compatibility
+Settings.outgoing_emails['enabled']           ||= true if Settings.outgoing_emails['enabled'].nil?
```

Comment:
No, it is consistent, see [below](https://github.com/jirutka/gitlabhq/blob/email-settings/config/initializers/1_settings.rb#L142-L145).